internal/ingest/fakecs_test.go

```diff
@@ -154,6 +289,18 @@ func (s *fakeCharmStore) entityInfo(ch params.Channel, id *charm.URL) (re *entit
 	if ch == params.NoChannel {
 		ch = params.StableChannel
 	}
+	var resources map[string][]int
+	if be := s.baseEntity(id); be != nil {
+		if ch == params.UnpublishedChannel {
+			panic("unimplemented: unpublished channel should get latest rev of all resources")
```

Comment:
should it? I'd have thought it should have no resources, but that shows what I know.